.github/workflows/test-linux-gpu.yml

```diff
@@ -0,0 +1,73 @@
+name: Unit-tests on Linux GPU
+
+on:
+  pull_request:
+  push:
+    branches:
+      - nightly
+      - main
+      - release/*
+  workflow_dispatch:
+
+env:
+  CHANNEL: "nightly"
+
+jobs:
+  tests:
+    strategy:
+      matrix:
+        python_version: ["3.8"]
+        cuda_arch_version: ["11.6"]
```

Comment:
They are dropping the support for 11.6, so might be better to move to 11.7.

https://pytorch.slack.com/archives/C2077MFDL/p1675256463971369

Reply:
Thanks for taking care of this in https://github.com/pytorch/text/pull/2040